### DISCUSSION TITLE
Why slider that have 4 point and remove 1 point the slider isn't like slider have 3 point not remove?

### DISCUSSION THREAD

**Discussion by timelessnesses** [ppy/osu#14177] [Q&A] [answered]
So here's really **sdfsnkjfjgskdbznvskfodfnkgdfg** thing in mapping
So you make a slider circle
![image](https://user-images.githubusercontent.com/59832159/128674260-bfecdd2d-48e2-4be6-8abc-b4d98540dae9.png)
But oops! You add 4 points!
![image](https://user-images.githubusercontent.com/59832159/128674296-3a63944a-f4a1-474f-8410-f8c3a1af8765.png)
But you think if I remove 1 point it should okay.
![image](https://user-images.githubusercontent.com/59832159/128674347-97734841-ad15-42e0-ae31-c9ee3efb1e19.png)
When I do it
![image](https://user-images.githubusercontent.com/59832159/128674377-2bd9a225-8c11-4cfe-a761-f7e1ffb488da.png)
**w h a t**

**Comment by peppy** [comment, marked as answer]
Wrong game version. This is also already fixed in lazer (the version you're reporting this issue against).

Please use **undo** to undo your operation.
